logging-log4j/src/main/java/com/cisco/oss/foundation/logging/transactions/TransactionLogger.java

```diff
@@ -167,6 +167,18 @@ public static String getFlowContextAsync(TransactionLogger logger) {
     return logger.flowContext;
   }
 
+  /**
+   * Add property to 'properties' map on transaction
+   * @param key - of property
+   * @param value - of property
+   */
+  public static void addProperty(String key, String value) {
+    TransactionLogger instance = getInstance();
+    if (instance != null) {
+      instance.properties.put(key, value);
```

Comment:
YES, if we want to be more dynamic and enable user to overwrite internal properties of logger, else NO.
I'll go for YES